citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -340,13 +335,28 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
 
     		List<Citation> citations = collection.getCitations();
     		String contentCollectionId = resource.getContainingCollection().getId();
-    		String exportUrlAll = collection.getUrl(org.sakaiproject.citation.api.CitationService.REF_TYPE_EXPORT_RIS_ALL) + "?citationCollectionId=" + citationCollectionId + "&resourceDisplayName=" + title + "&contentCollectionId=" + contentCollectionId;
+    		String exportParams =  "?resourceDisplayName=" + resource.getProperties().getProperty(ResourceProperties.PROP_DISPLAY_NAME) + "&resourceId=" + resource.getId();
+    		String exportUrlAll = fullCollection.getUrl(org.sakaiproject.citation.api.CitationService.REF_TYPE_EXPORT_RIS_ALL) + exportParams;
+    		String displayDate = null;
+    		try {
+    			displayDate = new SimpleDateFormat("dd/MM/yyyy").format(properties.getDateProperty(ResourceProperties.PROP_MODIFIED_DATE));
+    		} catch (EntityPropertyNotDefinedException e) {
+    			m_log.warn("CitationListAccessServlet.handleViewRequest() : Property name requested is not defined - for contentCollectionId" + contentCollectionId);
```

Comment:
It seems more important than info but less important than error - I don't mind changing it to different level.